### ISSUE TITLE
Units displayed on the charts are cut

### ISSUE THREAD

**Issue by oleiade** [grafana/xk6-dashboard#54]
This is something we discussed internally already:

![Screenshot 2023-08-28 at 16 30 46](https://github.com/grafana/xk6-dashboard/assets/690867/8ca32df5-0ab4-4ad0-92c3-4375f62f2e8b)
![Screenshot 2023-08-28 at 16 30 53](https://github.com/grafana/xk6-dashboard/assets/690867/57de0d08-0ae3-4adf-89bd-dcf8b2554fb2)

The units on either side of the charts are cut.

**Comment by szkiba** [comment]
Yes, I'm not satisfied with the current state, it is a bit better than prev, but far from perfect. (currently it is the limitation of the underlying time formatting lib, probably I should switch to other (but small) lib)
